src/net/http.rs

```diff
@@ -5464,12 +5467,13 @@ mod test {
             let e = http.read_payload(&preamble, &bad_content_length.as_bytes()[offset..]);
             let estr = format!("{:?}", &e);
 
-            assert!(e.is_err(), estr);
+            assert!(e.is_err(), "{}", estr);
```

Comment:
can simplify here

Reply:
Addressed in 93a7c393069ca4420433e03338a6534bee19dc8f